health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/FormulaConfigScreen.js

```
@@ -0,0 +1,90 @@
+            input: "input4",
+            operator: "+",
+            assumptionValue: "assumption4",
+            output: "output4",
+            showOnEstimationDashboard: true,
+            category: "CATEGORY_2"
+        }
+    ];
+
+    // You can log or manipulate the array as needed
+
+    let dic = {};
```

Biome

[error] This let declares a variable that is only assigned once.

'dic' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)